_posts/2023-02-22-user-defined-operators.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: post
+published: true
+title: 'Feature Deep-Dive: User-Defined Operators'
+date: '2023-02-22'
+author: Kamil Śliwak, Matheus Aguiar
+category: Explainers
+---
+
+The prominent new feature in [Solidity 0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19)
+is the support for defining *operators* on [user defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs).
+
+If you have not been keeping up with recent features, UDVTs are a new class of types introduced in Solidity 0.8.8.
+They provide zero-cost abstraction of an elementary value type that results in a completely new type.
```

Comment:
Personally, I'm not a fan of the weird rusticean "zero-cost abstraction" thing :-). Fine with keeping it, but I'd just say "an abstraction". But also fine with keeping it.

Reply:
That was actually requested by @NunoFilipeSantos in https://github.com/ethereum/solidity-blog/pull/202#discussion_r1085549651 so I just went with it and just adapted it. For me either way is fine. Changed to this now:
```
They provide an abstraction over an elementary value type that results in a completely new type.
```